miqa/core/tests/test_import.py

```diff
@@ -261,5 +261,5 @@ def test_import_s3_preserves_path(project_factory):
     frame = Frame.objects.first()
     assert (
         frame.raw_path
-        == 's3://miqa-sample/IXI_small/Guys/IXI002/0828-DTI/IXI002-Guys-0828-DTI-00.nii.gz'
+        == 's3:/miqa-sample/IXI_small/Guys/IXI002/0828-DTI/IXI002-Guys-0828-DTI-00.nii.gz'
```

Comment:
We need the double slash to be preserved, see my comment above.

Reply:
Ah ok, I thought it was a typo; I didn't understand the purpose of the test.

Reply:
Changed in [1f06c2d](https://github.com/OpenImaging/miqa/pull/318/commits/1f06c2dfd89ee714109bc9566158e6088da4b9cd)